clustering/common/src/main/java/org/jboss/as/clustering/controller/RequiredCapability.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2015, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.as.clustering.controller;
+
+/**
+ * Enumerates capability requirements for clustering resources
+ * @author Paul Ferraro
+ */
+public enum RequiredCapability implements Requirement {
```

Comment:
All clustering subsystem modules import wildfly-clustering-common, so all have access to this enum.  Why copy the same non-trivial string in multiple places when it can be externalized in 1 location?

Re: straying from what we're doing elsewhere - In general, all of the clustering subsystems stray from what we're doing elsewhere in many areas, including operation/metric handling, transformations, xml parsing/marshalling - so, in a way, this is in good company. :)